crates/core/types/transaction.rs

```diff
@@ -795,6 +840,40 @@ mod serde_impl {
             struct_serializer.end()
         }
     }
+
+    /// Unsigned Transaction struct generic to all types
```

Comment:
should this code live in `rpc` crate?

Reply:
I think it is more convenient to leave it in the core crate, as we will probably be using it in the future (implementing functionality or conversions) and we cannot import the rpc crate from core due to cyclic dependencies

Reply:
That's my question actually, where would we use this besides to serialize and deserialize RPC transactions? Because ideally we would only use the Transactions defined in core.